docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -243,16 +244,16 @@ The default cache uses a token as a key and each entry can have `TokenIntrospect
 Please experiment with the default cache implementation or register a custom one.
 
 [[jwt-claim-verification]]
-=== JSON Web Token Claim Verification
+=== JSON web token claim verification
```

Comment:
```suggestion
=== JSON Web Token claim verification
```

`JSON Web Token` is a common way to refer to JWT, `JSON web token` reads unusual :-)

Reply:
I've added [JSON Web Token](https://jwt.io/) to our list of proper nouns.